create/templates/beacon_extension/javascript.js

```diff
@@ -0,0 +1,12 @@
+shopify.extend(
```

Comment:
This file should be renamed to `src/index.js` since your extension config lists the main entrypoint as `main: src/index.js`. Alternatively, change the main entrypoint. I would go with the former as leaving the entrypoint blank will default it to `main: src/index.js`